frontend/src/components/Pages/Login.jsx

```
@@ -1,8 +1,14 @@
+
+import React, { useState , useEffect } from "react";
+import { Link } from "react-router-dom";
+import photo from "../../assets/login.png";
+
 import { Link, useNavigate } from "react-router-dom";
```

Biome

[error] Shouldn't redeclare 'Link'. Consider to delete it or rename it.

'Link' is defined here:


(lint/suspicious/noRedeclare)